.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "@babel/eslint-parser",
+    "parserOptions": {
+      "ecmaFeatures": {
+        "jsx": true
+      },
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb", "plugin:react/recommended"],
```

Comment:
- [ ] Your frontend application is connecting properly to the API backend however data is not able to be fetched randomly as required. This is likely due to improper configuration from your frontend. Kindly ensure that ESlint is properly configured using [this link](https://github.com/microverseinc/linters-config/tree/master/react-redux)

**Eslint error:**
![image](https://user-images.githubusercontent.com/12745474/146811330-aabe213c-e8f7-4ad8-9787-e7819d8febc6.png)

Reply:
Hi, @rloterh thanks! One question, the problem is for linters or the message displayed is always the same? I can't reproduce the error here locally and there are no linter errors with Github actions. 

Thanks again for your feedback

Reply:
Hi @kensayo 
Sorry, i should have broken this down.
The short of it is 
1. The message is not displaying randomly as required
2. ESLint is are not configured correctly. 
**Both are required**. However, I was anticipating that 2 led to 1.

However, kindly bear in mind that linters are very essential in the developers' space. 😄😄

Reply:
The application works without errors on my system. 
![Screenshot 2021-12-22 at 03-23-43 React App](https://user-images.githubusercontent.com/1642014/147014623-a02cc0dc-663d-4594-a272-905170d4940d.png)